packages/velo-external-db-commons/lib/data_commons.js

```diff
@@ -57,6 +57,19 @@ const AdapterOperators = {
     urlized: 'urlized'
 }
 
+const WixOperators = {
```

Comment:
We generate a random Wix operator in `gen.js`, so for the sake of order created a new enum, the random-wix-operator function takes a random value from this enum.